scan/src/utils/proof/NonEVMProof.re

```diff
@@ -265,34 +267,53 @@ let rec encode =
     }
   | BlockHeaderMerkleParts({
       versionAndChainIdHash,
-      timeHash,
+      timeSecond,
+      timeNanoSecond,
       lastBlockIDAndOther,
       nextValidatorHashAndConsensusHash,
       lastResultsHash,
       evidenceAndProposerHash,
     }) => {
-      Some(
-        JsBuffer.concat([|
-          versionAndChainIdHash,
-          timeHash,
-          lastBlockIDAndOther,
-          nextValidatorHashAndConsensusHash,
-          lastResultsHash,
-          evidenceAndProposerHash,
-        |]),
+      Obi.encode(
+        "{versionAndChainIdHash: bytes, timeSecond: u64, timeNanoSecond: u64, lastBlockIDAndOther: bytes, nextValidatorHashAndConsensusHash: bytes, lastResultsHash: bytes, evidenceAndProposerHash: bytes}/{_:u64}",
```

Comment:
timeNanoSecond should be u32 according to this implementation from the chain team https://github.com/bandprotocol/bandchain-contracts/blob/master/src/sol-v06/contracts/bridge/library/Utils.sol#L59